client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -353,6 +355,23 @@ export const ResponseLineEdit = ({
                   )}
                 </Box>
               </Box>
+              {hasApproval && (
+                <Box>
+                  <InputWithLabelRow
+                    Input={
+                      <NumericTextInput
+                        width={INPUT_WIDTH}
+                        value={draft?.approvedQuantity}
+                        disabled={!!hasLinkedRequisition}
+                        onBlur={save}
```

Comment:
Should always be disabled and shouldn't be editable by local user. This is all done through the Auth App (: